gamestonk_terminal/main_helper.py

```diff
@@ -136,7 +136,7 @@ def load(other_args: List[str], s_ticker, s_start, s_interval, df_stock):
         "-s",
         "--start",
         type=valid_date,
-        default="2019-01-01",
+        default="2020-01-01",
```

Comment:
Should we do like you did on one of the last PRs? 2 years prior to today?'

Reply:
lets do a year.  More than that seems overkill for visualizations (even the 1/1/20 seems overkill)

Reply:
Agreed! my suggestion was just to not have a default date, but a default "offset"